### ISSUE TITLE
Error when only single tab

### ISSUE THREAD

**Issue by bastengao** [brentvatne/react-native-scrollable-tab-view#61]
There is a example in below

```
<ScrollableTabView edgeHitWidth={deviceWidth / 2}>
  <View tabLabel="Only me">...</View>
</ScrollableTabView>
```

<img width="341" alt="2015-10-27 11 27 22" src="https://cloud.githubusercontent.com/assets/785335/10748706/d381cade-7c9d-11e5-8d70-f12e198c9d8b.png">

**Comment by ippa** [comment]
I got bitten by this as well #72 .. I think it's easily patchable.

**Comment by ericraio** [comment]
@brentvatne @ippa @bastengao I created a pull request to solve this, what do you think?

https://github.com/brentvatne/react-native-scrollable-tab-view/pull/98

I was thinking we should just throw an error message, why use this library for only one tab?

**Comment by ippa** [comment]
@ericraio @brentvatne : While testing / developing you might have just one tab from time to time:

- In my case, the first time I tested it out, I only created one tab and then had to dig around due to the crash
- While developing you might have situations where you comment out buggy/expensive tabs and end up with only one for a certain period of time, the app shouldn't crash then

I can't see any downside to make it work with only 1 tab. The fix is probably supersimple as I mention in #72.

If we do end up with an errormessage -- "child elements" might not mean something to certain users. How about "tabs" or "tab views" to use the term the libraryname itself uses.

@brentvatne : any final say? :)
